lib/engine/train.rb

```diff
@@ -78,6 +79,18 @@ def buyable
       @buyable && !@obsolete
     end
 
+    def local?
+      return @local unless @local.nil?
```

Comment:
!@local

Reply:
No since its a boolean you cant use !@local